lib/adapters/http.js

```diff
@@ -245,7 +245,9 @@ module.exports = function httpAdapter(config) {
 
     // Handle errors
     req.on('error', function handleRequestError(err) {
-      if (req.aborted && err.code !== 'ERR_FR_TOO_MANY_REDIRECTS') return;
+      if (req.aborted
+        && err.code !== 'ERR_FR_TOO_MANY_REDIRECTS'
+        && err.code !== 'ERR_FR_REDIRECTION_FAILURE') return;
```

Comment:
@RubenVerborgh Will `follow-redirects` add a common flag?

Reply:
Depends; what are all the possible error codes for aborted?

You could also `!/^ERR_FR_/.test(err.code)`.

Reply:
@RubenVerborgh Thanks for your quick response. You reminded me that axios doesn't need to throw every error from `follow-redirects`.

Emit handlers are called synchronously, right? Then we can ignore ERR_FR_MAX_BODY_LENGTH_EXCEEDED.

```
  // Error when we exceed the maximum body length
  else {
    this.emit("error", new MaxBodyLengthExceededError());
    this.abort();
  }
```

Reply:
In fact, I am not sure what happens if ERR_FR_REDIRECTION_FAILURE was thrown and who causes the abort. Because it was different with ERR_FR_TOO_MANY_REDIRECTS, which means the request was aborted definitely by follow-redirects.

Reply:
`ERR_FR_REDIRECTION_FAILURE` is emitted (asynchronously) when a redirected request fails (synchronously); the underlying error is in `.cause`.

Reply:
@iradul I come up with a new idea. The reason why axios ignores errors by aborted requests (I think) is that axios will abort the request if users cancelled the request by cancelToken or the timeout was triggered. How about only skipping errors for those 2 cases and always throw others?

Reply:
@chinesedfan I think only canceled request could be ignored since a user knows that he canceled it. I personally think it would still be nice to have rejection even in this case.

Everything else should have some sort of feedback, even timeout because otherwise, the user will be expecting something to happen and if an error is dropped answer will never come.

Reply:
@iradul Don't worry. Ignoring here (the commented line248) doesn't mean nothing is returned to users. For cancellation and timeout, they have their own handlers to reject an error.